docs/_documentations/vsc-firstproject.md

```diff
@@ -49,15 +49,13 @@ This icon launches your web browser and displays your application.
 
 Congratulations, you have now created, built and run your first Codewind project using Codewind on your local machine.
 
-# Next Steps
+## Next Steps
 
-For next steps, choose from the following:
+Choose from the following:
 
-* If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
+**Getting started with Codewind - next steps** Continue to instructions for [Making a Code change with VS Code](./vsc-codechange.html) to see the auto-build-and-run feature in action.
+
+**Installing Codewind for remote use** If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
```

Comment:
- "Prerequisite" doesn't need a hyphen.
- Could we remove the "to" after "proceed" to make the sentence complete?